common/config/.golangci.yaml

```diff
@@ -49,6 +50,15 @@ linters-settings:
   gocyclo:
     # minimal cyclomatic complexity to report
     min-complexity: 15
+  depguard:
+    rules:
+      main:
+        deny:
+          - pkg: "github.com/golang/protobuf"
+            desc: "Please use google.golang.org/protobuf instead."
+          - pkg: 'gopkg.in/yaml.v3'
```

Comment:
`gopkg.in/yaml.v2` and `github.com/ghodss/yaml` maybe import by accident, you should add them too.

Reply:
of course, i will solve it soon